ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/transactions/PendingTransactions.java

```diff
@@ -51,10 +56,11 @@
  */
 public class PendingTransactions {
 
-  public static final int MAX_PENDING_TRANSACTIONS = 4096;
-  public static final int DEFAULT_TX_RETENTION_HOURS = 13;
+  private static final Logger LOG = LogManager.getLogger();
+  public static final int MAX_PENDING_TRANSACTIONS = 1024;
+  public static final int DEFAULT_TX_RETENTION_SECONDS = 60;
```

Comment:
How did you decide on these constants?

Reply:
Arbitrarily. The idea is to reduce the max pending transactions so i divided the max size by 4. And for the retention period it was mentioned in the JI description. Adrian suggested to evict transactions older than 1 minute. By the way the two are configurable. I have worked on the default values because often they are not set.